### PR TITLE
[ImportCommunities] Add token models needed for CommunityIntroPopup

### LINKED ISSUE #11773: [ImportCommunities] Add token models needed for CommunityIntroPopup
### Description
Add
```
permissionsModel
walletAccountsModel
assetsModel
collectiblesModel
```
needed for CommunityIntroPopup when trying to import a community with a public key.

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #11773

### What does the PR do
ImportCommunities: Add token models needed for CommunityIntroPopup

### Affected areas
ImportCommunities: community intro popup

### CODE REVIEW

File: ui/app/mainui/Popups.qml [hunk 1]
```diff
@@ -537,7 +537,8 @@ QtObject {
                         communityIntroDialog.close();
                     }
                 }
-                onClosed: { destroy(); }
+                onOpened: { root.rootStore.communityKeyToImport = communityIntroDialog.communityId; }
```
Comment (caybro) [inline ui/app/mainui/Popups.qml:540]: I'd just change this to `onAboutToShow`

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/mainui/Popups.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/mainui/Popups.qml]: done

File: ui/app/AppLayouts/stores/RootStore.qml [hunk 2]
```diff
@@ -41,6 +50,20 @@ QtObject {
...
+            AnyOf {
+                // We accept tokens from this community or general (empty community ID)
+                ValueFilter {
+                    roleName: "communityId"
+                    value: ""
+                }
+
+                ValueFilter {
+                    roleName: "communityId"
+                    value: root.communityKeyToImport
```
Comment (jrainville) [inline ui/app/AppLayouts/stores/RootStore.qml:63]: FYI, this needs to be the public key

Comment (alexandraB99) [reply ui/app/AppLayouts/stores/RootStore.qml:63]: yes, is `root.joinCommunity(d.publicKey, d.communityDetails);` and then we get `0x02c6a9e66f0153f9eaac2a25e2f8c343da5c31dcd6b2ed10f5c4350f4c67709248`

Comment (alexandraB99) [reply ui/app/AppLayouts/stores/RootStore.qml:63]: @jrainville is this ok?

**Review by alexjba — APPROVED** [review]
LGTM!

**Review by noeliaSD — APPROVED** [review]
LGTM!

**Pull request merged**